src/mono/System.Private.CoreLib/src/System/Reflection/Emit/EnumBuilder.Mono.cs

```diff
@@ -209,6 +209,10 @@ public override Type GetEnumUnderlyingType()
             return _underlyingType;
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2110:RequiresDynamicallyAccessedMembers",
+            Justification = "For instance member internal calls, the linker preserves all fields of the declaring type. " +
+            "The _tb field has DynamicallyAccessedMembersAttribute requirements, but the field access is safe because " +
```

Comment:
This is a super weird place for this suppression IMO. You don't even see the `_tb` field around here.

Reply:
I filed https://github.com/mono/linker/issues/2188 about the warning origin.